### DISCUSSION TITLE
Please make a course and teach us this amazing IDE

### DISCUSSION THREAD

**Discussion by fisforfaheem** [flet-dev/flet#1409] [Ideas]
I love this, but i dont know python.i only know dart. i think most people can benefit

**Comment by ndonkoHenri** [comment]
Learn python and you will be good to go. Then you proceed by having a deep look at docs and also the great amount of online articles/blogposts and videos on flet out there. 
You are just a google search away from them.

Please avoid creating unnecessary issues. Create discussions instead.

**Reply by fisforfaheem** [reply]
Could you please share any resource, I am a beginerr flutter dEV and your apps for windows just made me crazy fall in love, just saw, bing rewards app on github by ur framework its maazing
